frontend/src/App.tsx

```diff
@@ -1,33 +1,60 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from 'react';
+// import './App.css'
 
-function App() {
-  const [count, setCount] = useState(0)
+const App = () => {
+  const [base64Image, setBase64Image] = useState<string | null>(null);
+  const handleInputFile = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[0];
+    if(file) {
+      const reader = new FileReader();
```

Comment:
ここは先にfiles変数に格納してfilesの要素数を使用して判定させた方が良いかと思います

Reply:
`files`が配列であることと、長さが1以上であることを検証した方がいいかな
```suggestion
    if(!Array.isArray(files) || files.length < 1) {
      return;
    }
```